modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -0,0 +1,64 @@
+= Secure View
+
+When Collabora Online is properly setup and integrated into ownCloud Server (10.2 or above) Secure View functionality is available.
+Secure View is a mode where users can place limitations on files and folders that are shared. 
+These limitations include:
+[[secure-view-mode-limitations]]
+* No copying
+* No downloading
+* No editing
+* Watermarking with optional print and export
++
+NOTE: Administrators can configure the watermark text.
+* Printing and exporting to PDF with watermarks included
+
+IMPORTANT: Documents never leave the server when shared with Secure View.
+Collabora Online Server opens them and streams the files to the user's browser (much like a video stream).
+Consequently, there's no way to extract the original document from the browser.
+
+== Supported File Formats
+
+Secure View only supports a limited number of file formats; these are:
+
+* Microsoft Word (.docx)
+* Microsoft Excel (.xlsx)
+* Microsoft PowerPoint (.pptx)
+* OpenDocument Text Document (.odt)
+* OpenDocument Presentation Document (.odp)
+* OpenDocument Spreadsheet Document (.ods)
+* PDF
+
+If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible in any way.
+
+== Prerequisites
+
+* ownCloud 10.2 or higher _Enterprise Edition_
+* Collabora Online set up and integrated
+
+NOTE: This functionality does not work with Public Links.
+
+== How to Enable Secure View
+
+To enable _Secure View_, navigate to menu:Settings[Admin > Additional (Admin) > Collabora Online].
+At the bottom of the Collabora Online section, enable btn:[Enable Secure View].
+
+Once enabled, default share permissions for all users can, optionally, be enabled. 
+Currently, these default share permissions are:
+
+* *Secure View (with watermarks)*. 
+   When enabled, files are shared in Secure View mode. In this mode , all xref:secure-view-mode-limitations[the Secure View limitations] are in-effect. 
```

Comment:
To add: `Secure View (with watermarks)` is mutually exclusive with `can edit`. Enabling one of them will disable the other.